python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -620,3 +620,17 @@ def test_norm_failure_error_message(lmdb_version_store_v1):
                [write_exception, batch_write_exception])
     assert all("pickle_on_failure" not in str(e.value) for e in
                [append_exception, batch_append_exception, update_exception])
+
+def test_bools_are_pickled(lmdb_version_store_allows_pickling):
+    lib = lmdb_version_store_allows_pickling
+    sym = "test_bools_are_pickled"
+
+    df = pd.DataFrame({"a": [True, False]})
+    lib.write(sym, df)
+    lib.get_info(sym)['type'] == 'pickled'
+    assert_frame_equal(df, lib.read(sym).data)
+
+    df = pd.DataFrame({"a": [True, False, np.nan]})
```

Comment:
What's the behaviour if the `NaN` is the first element in the column? Is it the same as `arcticc`?

Reply:
If np.nan is first it gets converted to float True -> 1.0 False -> 0.0. It's the same in arcticc and ArcitcDB